src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -260,6 +260,7 @@ private void UpdateOrganelleUnlockTooltips()
             var tooltip = GetSelectionTooltip(organelle.InternalName, "organelleSelection");
             if (tooltip != null)
             {
+                tooltip.CanBeModified = organelle.CanBeModified;
```

Comment:
Is this change actually required considering the code in `SetupMicrobePartSelections`?
If this is needed, then I think it might warrant renaming this method to something like `UpdateOrganelleTooltips` (as long as there isn't already a method with basically that name).